src/TcBlackTests/VariableDeclarationTests.cs

```diff
@@ -179,7 +179,7 @@ private void AssertEquals(TcDeclaration expected, TcDeclaration actual)
             "deviceDown      AT     %QX0.2  :        BOOL ; ", 
             "deviceDown AT %QX0.2 : BOOL;"
         )]
-        [InlineData("devSpeed:TIME:=T#10ms;", "devSpeed : TIME := T#10ms;")]
+        [InlineData("devSpeed:TIME:=T#10ms;", "devSpeed : TIME := T#10MS;")]
```

Comment:
Very unfortunate that TwinCAT auto capitalize, converts the time units to upper case. The one with small case units is much better to read, because of the size difference of numbers and small case letters in most fonts. Since TwinCAT does this, I guess we'll stick with this.